clinica/pipelines/statistics_volume/statistics_volume_pipeline.py

```diff
@@ -0,0 +1,347 @@
+        datasink.inputs.base_directory = join(self.caps_directory, relative_path)
+
+        datasink.inputs.parameterization = True
+        if self.parameters['smoothing']:
+            datasink.inputs.regexp_substitutions = [
+                # t-stat map
+                (join(self.caps_directory, relative_path) + r'/spm_results_analysis_./(.*)',
+                 join(self.caps_directory, relative_path) + r'/\1'),
+
+                # contrasts
+                (join(self.caps_directory, relative_path) + r'/contrasts/(.*)',
+                 join(self.caps_directory, relative_path) + r'/\1'),
+
+                # resels per voxels
+                (join(self.caps_directory, relative_path) + '/resels_per_voxels/resels_per_voxel.nii',
+                 join(self.caps_directory, relative_path) + '/group-' + self.parameters['group_id'] + '_contrast-' + self.parameters['contrast'] + '_RPV.nii'),
+
+                # mask
+                (join(self.caps_directory, relative_path) + '/mask/included_voxel_mask.nii',
+                 join(self.caps_directory, relative_path) + '/group-' + self.parameters['group_id'] + '_contrast-' + self.parameters['contrast'] + '_mask.nii'),
+
+                # variance of error
+                (join(self.caps_directory, relative_path) + '/variance_of_error/(.*)',
+                 join(self.caps_directory, relative_path) + r'/\1'),
+
+                # tsv file
+                (join(self.caps_directory, relative_path) + r'/tsv_file/.*',
+                 join(self.caps_directory, relative_path) + '/' + pardir + '/group-' + self.parameters['group_id'] + '_participants.tsv'),
+
+                # report (figures)
+                (join(self.caps_directory, relative_path) + r'/figures/(.*)',
+                 join(self.caps_directory, relative_path) + r'/\1'),
+
+                # regression coefficient
+                (join(self.caps_directory, relative_path) + r'/regression_coeff/(.*).nii',
+                 join(self.caps_directory, relative_path) + '/group-' + self.parameters['group_id'] + '_contrast-'
+                 + self.parameters['contrast'] + '_measure-' + r'\1' + '_fwhm-' + str(self.parameters['smoothing']) + '_regressionCoefficient.nii'),
+
+            ]
+
+        datasink.inputs.tsv_file = self.tsv_file
+
+        self.connect([
+            (self.output_node, datasink, [('spmT_0001', 'spm_results_analysis_1')]),
+            (self.output_node, datasink, [('spmT_0002', 'spm_results_analysis_2')]),
+            (self.output_node, datasink, [('new_figure_names', 'figures')]),
+            (self.output_node, datasink, [('variance_of_error', 'variance_of_error')]),
+            (self.output_node, datasink, [('resels_per_voxels', 'resels_per_voxels')]),
+            (self.output_node, datasink, [('mask', 'mask')]),
+            (self.output_node, datasink, [('regression_coeff', 'regression_coeff')]),
+            (self.output_node, datasink, [('contrasts', 'contrasts')])
+        ])
+
```

Comment:
From what I read, the pipeline does not save intermediate results in the CAPS directory. Can you confirm?

Reply:
Nothing is saved in the caps until the very end of the pipeline (datasink node)